src/resources/postcss/components/_navigation.pcss

```diff
@@ -108,7 +119,7 @@
 
 			@media (--viewport-medium) {
 				background-image: svg-inline(caret-left);
-				height: 16px;
+				height: 14px;
 				margin-right: 15px;
 				width: 10px;
```

Comment:
small thing, but let's fix these widths to match the aspect ratio of the icon on a 14px height.